unittest/DAQModule_test.cxx

```diff
@@ -69,16 +70,16 @@ BOOST_AUTO_TEST_CASE(Commands)
 {
   daqmoduletest::GoodDAQModule gdm("command_test");
 
-  BOOST_REQUIRE(gdm.has_command("stuff"));
+  BOOST_REQUIRE(gdm.has_command("stuff", "RUNNING"));
   auto valid_commands = gdm.get_commands();
   BOOST_REQUIRE_EQUAL(valid_commands.size(), 1);
   BOOST_REQUIRE_EQUAL(valid_commands[0], "stuff");
 
   dunedaq::opmonlib::InfoCollector ic;
   gdm.get_info(ic, 0);
 
-  gdm.execute_command("stuff", {});
-  BOOST_REQUIRE_THROW(gdm.execute_command("other_stuff", {}), UnknownCommand);
+  gdm.execute_command("stuff", "RUNNING", {});
+  BOOST_REQUIRE_THROW(gdm.execute_command("other_stuff", "RUNNING", {}), UnknownCommand);
```

Comment:
There should also be unit tests for the InvalidState throw, as well as the "ANY" state matching.